### PR TITLE
Update github.com/avast/retry-go to v4.5.0

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

File: pkg/exporters/stackdriver/stackdriver_exporter.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 	"google.golang.org/api/option"
 	"k8s.io/klog/v2"
 
-	"github.com/avast/retry-go"
+	"github.com/avast/retry-go/v4"
 	"k8s.io/node-problem-detector/pkg/exporters"
```
Comment (MartinForReal) [inline pkg/exporters/stackdriver/stackdriver_exporter.go:34]: I'm wondering if import statements can be sorted using goimports

Comment (hakman) [reply pkg/exporters/stackdriver/stackdriver_exporter.go:34]: Probably not so hard. There's also a need to verify during automated testing.

**Pull request merged**